internal/provider/resource_cloudflare_teams_list.go

```diff
@@ -73,6 +73,7 @@ func resourceCloudflareTeamsListRead(ctx context.Context, d *schema.ResourceData
 	listItems, _, err := client.TeamsListItems(ctx, cloudflare.TeamsListItemsParams{
 		AccountID: accountID,
 		ListID:    d.Id(),
+		PaginationOptions: cloudflare.PaginationOptions{ PerPage: 1000 },
```

Comment:
this is a step in the right direction but doesn't account for when more than 1000 items are available. ideally, we need to paginate all available pages instead.

Reply:
i've added the iteration via e1a3384 so we should be right with large list items now.

Reply:
Thanks Jacob this makes a lot more sense than the hardcoded default. I was unable to get to this Friday and appreciate the assist! 💯